pygly/symbolstable.ini

```diff
@@ -473,8 +473,44 @@ IUPACSym    = Pent
 MassSym     = Pent
 SuperClass  = SuperClass.PENT
 
-[Sia]
+[NeuAc]
 IUPACSym    = NeuAc
 SuperClass  = SuperClass.NON
 Mods        = Mod.a Mod.keto Mod.d
 Subst       = Substituent.nAcetyl
+
+[D-Mannose Acid]
+LinCodeSym  = M
+LinCodeRank = 5
```

Comment:
Do not provide fields you don't know the value for. Please remove LinCode* Glycam* Mass*. The only one that should be here (and the others) is IUPACSym and those necessary to identify the entry.